app/filtering.js

```diff
@@ -571,7 +581,10 @@ module.exports.isResourceEnabled = (resourceName, url) => {
   }
 
   if ((resourceName === appConfig.resourceNames.ADBLOCK ||
-       resourceName === appConfig.resourceNames.TRACKING_PROTECTION)) {
+      appConfig[resourceName] &&
```

Comment:
It was there before, but has unnecessary double parens
if((condition === value)) { }